### PR TITLE
[4693] Change the FreeMarker import structure - bps update

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4693

### CODE REVIEW

File: src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/templates/web/pages/article.ftl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<#import "/templates/system/common/ice.ftl" as studio />
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/templates/web/pages/article.ftl:1]: 👇🏻

File: src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/templates/web/pages/category-landing.ftl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<#import "/templates/system/common/ice.ftl" as studio />
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/templates/web/pages/category-landing.ftl:1]: 👇🏻

File: src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/templates/web/pages/home.ftl [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<#import "/templates/system/common/ice.ftl" as studio />
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/templates/web/pages/home.ftl:1]: 👇🏻

**Review by rart — CHANGES_REQUESTED** [review]
Please revisit the indentation of files marked with "👇🏻" below

File: src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/ui.xml [hunk 4]
```diff
@@ -109,9 +109,10 @@
                                 <configuration>
                                     <title id="GlobalMenu.EncryptionToolEntryLabel" defaultMessage="Encryption Tool"/>
                                     <icon id="@material-ui/icons/LockOutlined" />
-                                    <widget id="craftercms.components.LegacySiteToolsFrame">
+                                    <widget id="craftercms.components.SiteEncryptTool">
                                         <configuration>
-                                            <tool>encrypt-tool</tool>
+                                            <logType>preview</logType>
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/ui.xml]: The only component that should be getting this prop is the `LogConsole` (`craftercms.components.LogConsole`)

**Review by rart — CHANGES_REQUESTED** [review]
- Please apply comments to all blueprint PRs
- Doesn't look like indentation is fixed on the previously pointed files

File: src/main/webapp/repo-bootstrap/global/blueprints/4000_empty/templates/web/entry.ftl [hunk 5]
```diff
@@ -1,30 +1,35 @@
...
+		<@crafter.head/>
+	</head>
+	<body>
+		<@crafter.body_top/>
+		<main>
+			<@crafter.h1 $field="title_t">${model.title_t}</@crafter.h1>
+			<@crafter.tag $field="body_html">${model.body_html}</@crafter.tag>
+		</main>
+
+		<script src="/static-assets/js/jquery.core.js"></script>
```
Comment (rart) [inline src/main/webapp/repo-bootstrap/global/blueprints/4000_empty/templates/web/entry.ftl]: jquery.js & crafter-yada-yada.js where removed in my PR to empty bp. looks like you favoured "yours" on during merge.

see https://github.com/craftercms/studio/pull/2592/files

**Review by rart — CHANGES_REQUESTED** [review]

File: src/main/webapp/repo-bootstrap/global/blueprints/4000_empty/templates/web/entry.ftl (hunk 5 above)
Comment (jvega190) [reply src/main/webapp/repo-bootstrap/global/blueprints/4000_empty/templates/web/entry.ftl]: Updated, removed import

**Review by rart — APPROVED** [review]

**Pull request merged**